pkg/operator/ceph/disruption/nodedrain/reconcile.go

```diff
@@ -127,19 +127,32 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 	}
+
+	controllerBool := false
+	blockOwnerDeletionBool := false
 	// Create or Update the deployment default/foo
 	deploy := &appsv1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      k8sutil.TruncateNodeName(fmt.Sprintf("%s-%%s", CanaryAppName), nodeHostnameLabel),
 			Namespace: r.context.OperatorNamespace,
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					APIVersion:         node.APIVersion,
+					Kind:               node.Kind,
+					UID:                node.GetUID(),
+					Name:               node.GetName(),
+					Controller:         &controllerBool,
```

Comment:
I believe you don't need to set `Controller` or `BlockOwnerDeletion` if they are the default value of `false`.

Reply:
+1 will trim

Reply:
(after testing though. BlockOwnerDeletion has a documented default of false, but Controller does not have a documented default)